### PR TITLE
[Blog] Building a Kimball dimensional model

### PR DESCRIPTION
**Pull request by jonathanneo**
## What are you changing in this pull request and why?

I am writing this blog post as part of the [dbt technical writing mentorship program](https://www.getdbt.com/blog/technical-writing-mentorship-program/). 

The motivation behind this blog post is that there aren’t many resources that teach analytics engineers how to create dimensional models using dbt. 

This tutorial aims to solve this by providing the definitive guide to dimensional modelling with dbt. 

Most of the resources in this article had to be created from scratch since there aren’t many guides and tutorials that I could find online. 

Deploy preview here: https://deploy-preview-3210--docs-getdbt-com.netlify.app/blog/kimball-dimensional-model 

## Checklist

- [x] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [About versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) so my content adheres to these guidelines.
- [x] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

### CODE REVIEW

**Review by kning — DISMISSED** [review]

**Review by kning — DISMISSED** [review]

File: website/blog/2023-04-18-building-a-kimball-dimensional-model-with-dbt.md [hunk 1]
```diff
@@ -0,0 +1,712 @@
...
+## Learning resources
+
+- [Kimball group learning resources](https://www.notion.so/Building-a-Kimball-dimensional-model-with-dbt-7fc6e04e26154c7baf67f695ae2a1ae2)
+- [The Data Warehouse toolkit book](https://www.kimballgroup.com/data-warehouse-business-intelligence-resources/books/data-warehouse-dw-toolkit/)
+- [Dbt discourse on whether dimensional modelling is still relevant](https://discourse.getdbt.com/t/is-kimball-dimensional-modeling-still-relevant-in-a-modern-data-warehouse/225)
+- [Dbt glossary on dimensional modelling](https://docs.getdbt.com/terms/dimensional-modeling)
+
+## Let's talk 
+
+The materials in this article were created by [Data Engineer Camp](https://dataengineercamp.com/), a 16-week data engineering bootcamp for professionals looking to transition to data engineering and analytics engineering.
```
Comment (KiraFuruichi) [inline website/blog/2023-04-18-building-a-kimball-dimensional-model-with-dbt.md]: Hey @jonathanneo this is amazing!! Do you mind moving this paragraph under your author's note below? Thanks!

Comment (jonathanneo) [reply website/blog/2023-04-18-building-a-kimball-dimensional-model-with-dbt.md]: Hey @KiraFuruichi this is now done :)

File: website/blog/2023-04-18-building-a-kimball-dimensional-model-with-dbt.md [hunk 2]
```diff
@@ -0,0 +1,708 @@
...
+
+<Lightbox src="/img/blog/2023-04-18-building-a-kimball-dimensional-model-with-dbt/dbt-dag.png" width="85%" title="Final dbt DAG"/>
+
+Congratulations, you have reached the end of this tutorial. If you want to learn more, please see the learning resources below on dimensional modelling. 
+
+## Learning resources
+
+- [Kimball group learning resources](https://www.notion.so/Building-a-Kimball-dimensional-model-with-dbt-7fc6e04e26154c7baf67f695ae2a1ae2)
+- [The Data Warehouse toolkit book](https://www.kimballgroup.com/data-warehouse-business-intelligence-resources/books/data-warehouse-dw-toolkit/)
+- [Dbt discourse on whether dimensional modelling is still relevant](https://discourse.getdbt.com/t/is-kimball-dimensional-modeling-still-relevant-in-a-modern-data-warehouse/225)
```
Comment (KiraFuruichi) [inline website/blog/2023-04-18-building-a-kimball-dimensional-model-with-dbt.md]:
```suggestion
- [dbt discourse on whether dimensional modelling is still relevant](https://discourse.getdbt.com/t/is-kimball-dimensional-modeling-still-relevant-in-a-modern-data-warehouse/225)
```

File: website/blog/2023-04-18-building-a-kimball-dimensional-model-with-dbt.md [hunk 3]
```diff
@@ -0,0 +1,708 @@
...
+<Lightbox src="/img/blog/2023-04-18-building-a-kimball-dimensional-model-with-dbt/dbt-dag.png" width="85%" title="Final dbt DAG"/>
+
+Congratulations, you have reached the end of this tutorial. If you want to learn more, please see the learning resources below on dimensional modelling. 
+
+## Learning resources
+
+- [Kimball group learning resources](https://www.notion.so/Building-a-Kimball-dimensional-model-with-dbt-7fc6e04e26154c7baf67f695ae2a1ae2)
+- [The Data Warehouse toolkit book](https://www.kimballgroup.com/data-warehouse-business-intelligence-resources/books/data-warehouse-dw-toolkit/)
+- [dbt discourse on whether dimensional modelling is still relevant](https://discourse.getdbt.com/t/is-kimball-dimensional-modeling-still-relevant-in-a-modern-data-warehouse/225)
+- [Dbt glossary on dimensional modelling](https://docs.getdbt.com/terms/dimensional-modeling)
```
Comment (KiraFuruichi) [inline website/blog/2023-04-18-building-a-kimball-dimensional-model-with-dbt.md]:
```suggestion
- [dbt glossary on dimensional modelling](https://docs.getdbt.com/terms/dimensional-modeling)
```

**Review by KiraFuruichi — APPROVED** [review]

**Pull request merged**